launchUI.py

```diff
@@ -1566,8 +1566,9 @@ def place_xbar_options(master):
 def create_param_pannel(master):
     paramPannel = tkinter.LabelFrame(master, text='Configuration', bd=BORDER, relief='groove')
     paramPannel.grid(row=0, column=1, rowspan=1, columnspan=1, sticky="nsew")
+
     # Use columnconfigure and rowconfigure to partition the columns, so that each column and row will fill the corresponding space

```

Comment:
nit: period is missing at the end of comment.

Reply:
Okay. I will add that period.